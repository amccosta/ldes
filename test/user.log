612 18 400 0.0359552
553
561
