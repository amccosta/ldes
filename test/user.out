/* $Id: user.c,v 17.19 1999/02/16 01:25:08 ingber Exp ingber $ */

start:time: 0h 0m 0s; incr: 0h 0m 0s
exit code = 1
final cost value =  -0.03595517
parameter	value
0		         553
1		         561
end:time: 0h 0m 0.568035s; incr: 0h 0m 4.33681e-17s
